make/modules/jdk.incubator.vector/Lib.gmk

```diff
@@ -39,4 +39,17 @@ ifeq ($(call isTargetOs, linux windows)+$(call isTargetCpu, x86_64)+$(INCLUDE_CO
   TARGETS += $(BUILD_LIBJSVML)
 endif
 
+ifeq ($(call isTargetOs, linux)+$(call isTargetCpu, aarch64)+$(INCLUDE_COMPILER2)+$(ENABLE_LIBSLEEF), true+true+true+true)
+  $(eval $(call SetupJdkLibrary, BUILD_LIBVMATH, \
+      NAME := vmath, \
+      CFLAGS := $(CFLAGS_JDKLIB) $(LIBSLEEF_CFLAGS) -fvisibility=default, \
```

Comment:
Why `-fvisibility=default`? (Sorry, only noticed this now)

Reply:
Yeah. Considering all the symbols in this lib are global and need to be exported, I added this flag here instead of the source code. I'v removed this in latest commit, and added the attribute visibility in source code like other jdk code. Please help to review again. Thanks a lot!